packages/block-editor/src/hooks/layout.js

```diff
@@ -35,23 +35,21 @@ const layoutBlockSupportKey = '__experimentalLayout';
 /**
  * Generates the utility classnames for the given block's layout attributes.
  *
- * @param { Object } block Block object.
+ * @param { Object } layout    Layout object.
+ * @param { string } blockName Block name.
  *
  * @return { Array } Array of CSS classname strings.
  */
-export function useLayoutClasses( block = {} ) {
+export function useLayoutClasses( layout, blockName ) {
```

Comment:
Why does the signature need to be changed, and can we retain `block` as one of the params? There haven't been many changes to this function in a long time, but from memory, one of the reasons to include the full block object was so that its easier to extend the function to include classnames for things that might not live in `layout` (like classnames related to which spacing values are set). Arguably if we're going to do that, we might have a parent `useSemanticClassNames` hook that then calls this hook, so might not be a deal-breaker 🙂

Reply:
Thanks for the update here, I like the consistent shape between `useLayoutClasses` and `useLayoutStyles` now 👍